lib/bootloader_setup.pm

```diff
@@ -356,23 +359,35 @@ sub bootmenu_default_params {
         # gfxpayload variable replaced vga option in grub2
         if (!is_jeos && !is_caasp && (check_var('ARCH', 'i586') || check_var('ARCH', 'x86_64'))) {
             type_string_slow "vga=791 ";
-            type_string_slow "video=1024x768-16 ";
-            assert_screen check_var('UEFI', 1) ? 'inst-video-typed-grub2' : 'inst-video-typed', 4;
+            if (check_var("INSTALL_TO_OTHERS", 1) || !$args{pxe}) {
```

Comment:
it is up to you, but I would prefer :
```
my $videomode = (check_var("INSTALL_TO_OTHERS", 1) || !$args{pxe} ? "video=1024x768-16" : "xvideo=1024x768 ";
type_string_slow $videomode;
```

Reply:
I avoid one-liners when condition is more complex than one check.

Reply:
make sense